### PR TITLE
Add a Getting Involved guide

### PR DESCRIPTION
**Pull request by bnb**
This adds a very bare-bones guide to getting involved to the Community Committee. The contents reference an Issue (https://github.com/nodejs/community-committee/issues/74) and a PR (https://github.com/nodejs/community-committee/pull/73) - that I've submitted as easy wins for things to help people get started.

This is *very* much a WIP, and any + all input is appreciated on how to improve. ❤️

### CODE REVIEW

**Comment by renrutnnej** [comment]
I'm curious, how is this different from the collaborator guide? And couldn't we consolidate this info into one doc?

**Comment by nebrius** [comment]
I agree with @renrutnnej, this should go in CONTRIBUTING.md

**Comment by bnb** [comment]
@renrutnnej That is actually a problem I ran into at the beginning - I was having a hard time distinguishing between what should go in a Contributing Guide versus something that's completely introductory for newcomers. 

The distinction I landed on was that the Collaborator Guide seemed to be a reference _for existing collaborators_ (which, I just learned, is actually distinct from members . The goal with this was to have a starting point where we can directly point to for people who are curious about how to get involved, but aren't sure exactly how.

That could possibly fit within the scope of the Collaborator Guide as a section - would love to hear your thoughts 👍

**Comment by renrutnnej** [comment]
I’m concerned that we're putting a lot of effort into documentation and process before we’ve had solid examples to document. 

My experience with onboarding new folks in the Hoodie project showed me that you want to make documentation as simple & concise as humanly possible to empower action and not let people get confused and disengage.

**Comment by nebrius** [comment]
> The distinction I landed on was that the Collaborator Guide seemed to be a reference for existing collaborators

Not really, if it comes off that way, then that's a problem in the contributing guide.

These changes definitely should go in the CONTRIBUTING.md. That file is special in GitHub land because it's what's referenced in issues and pull requests in the GitHub UI. We definitely don't want to loose that ability.

**Comment by bnb** [comment]
>My experience with onboarding new folks in the Hoodie project showed me that you want to make documentation as simple & concise as humanly possible to empower action and not let people get confused and disengage.

Awesome feedback, @renrutnnej. 🙏 

If there's a more simple way to get people onboarded, I'm 100000% for that - I really just want to be able to empower people to get started, and at this point I'm jut not quite sure how we can do that consistently and effectively. That was my goal with this - happy to take an alternative route.

File: GETTING_INVOLVED.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Getting Involved with the Node.js Community Committee
+
+## Start Here:
+
+- Take a look at any Issues tagged with [Good First Contribution]()
+
+## Teams and Working Groups
+The Community Committee is a Committee that represents a collection of teams and working groups that are expressly working toward growing the Node.js Community in via contributions that are not necessarily technical in nature, but rather focus on the community that uses Node.js.
```
Comment (mhdawson) [inline GETTING_INVOLVED.md]: How about something like this: ?
```
The Community Committee is a Committee that represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. These teams and working groups need a broad range of skills and allow you to offer contributions that are not necessarily technical in nature.
```

**Comment by mhdawson** [comment]
Is it also worth mentioning that watching the recordings of the Community Committee meetings and reviewing the minutes is also a good way to get up to speed with what is being discussed ?

**Comment by pup** [comment]
@mhdawson i think that's a good idea! this shouldn't be limited to only commcomm though - it should say something like `Read through meeting notes of a team that interests you` or sometyhing

**Comment by bnb** [comment]
@mhdawson Issue right now is there's not an easy way to go find the meeting recordings. Minutes are in place, I'll likely go submit a PR to add recording URLs to the minutes.

@pup Also +1! I want to be sure we have that content before adding it though - don't want to direct people to something that doesn't exist yet 😊

**Comment by hackygolucky** [comment]
@nebrius Are you +1 on this document now but -1 on its location? If so, will moving it to CONTRIBUTING.MD change your mind?

**Comment by nebrius** [comment]
@hackygolucky not yet. Once this is moved to CONTRIBUTING.md though, I think we'll be ready to go!

**Comment by bnb** [comment]
@nebrius / @hackygolucky moved to `CONTRIBUTING.md` - didn't realize that was the blocker 😅 

Please LMK if the location is good or bad - I figured "Getting Started" should go at the beginning, but can move as needed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,6 +11,18 @@ expectations the Node Foundation requires of its contributors.
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
 
+# Getting Involved with the Node.js Community Committee
```
Comment (nebrius) [inline CONTRIBUTING.md]: Looking at this holistically within the file, I think we should drop the title here, and make "Start Here" and "Teams and Working Groups" at the same level as "Logging Issues". As is, this title feels kinda redundant to me.

Comment (RichardLitt) [inline CONTRIBUTING.md]: Agreed.

**Comment by bnb** [comment]
@nebrius Updated 😁

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
```
Comment (RichardLitt) [inline CONTRIBUTING.md]: Subset makes it sound a bit too logical, and unnecessary. I would go with `A collaborator is a contributor who has been given write access to the repository.`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
...
 
+## Start Here:
+
+- Take a look at any Issues tagged with [Good First Contribution](https://github.com/nodejs/community-committee/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+contribution%22)
```
Comment (RichardLitt) [inline CONTRIBUTING.md]: Add a period at the end. Any reason Issues is capitalized? It doesn't need to be.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
...
+
+## Teams and Working Groups
+The Community Committee is a Committee that represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. These teams and working groups need a broad range of skills and allow you to offer contributions that are not necessarily technical in nature.
```
Comment (RichardLitt) [inline CONTRIBUTING.md]: "The Community Committee represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. Node.js as a whole needs a broad range of skills and contributions that are not necessarily technical in nature to thrive."

The teams and working groups don't need skills, and don't actually allow contributions. Tightened up the language a bit.

File: CONTRIBUTING.md (hunk 4 above)
Comment (bnb) [inline CONTRIBUTING.md]: I typically capitalize Issues because it's a name of a tool/feature in GitHub (proper noun), but that's just personal bias. I can change it if you think it's more appropriate 👍

File: CONTRIBUTING.md (hunk 3 above)
Comment (bnb) [inline CONTRIBUTING.md]: Can this be submitted in another PR? These lines are not a part of this PR, and I'd like to focus the edits in this PR to the updates presented around the Getting Started guide 😅

File: CONTRIBUTING.md (hunk 4 above)
Comment (RichardLitt) [inline CONTRIBUTING.md]: Nah, that works. Just curious.

File: CONTRIBUTING.md (hunk 5 above)
Comment (bnb) [inline CONTRIBUTING.md]: Tightened it up a bit more, as the last few words were a bit confusing.

```
The Community Committee represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. Node.js as a whole needs a broad range of skills and contributions, which are not necessarily technical in nature, to thrive.
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (RichardLitt) [inline CONTRIBUTING.md]: See https://github.com/nodejs/community-committee/pull/94.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
...
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
 
+## Start Here:
+
+- Take a look at any issues tagged with [Good First Contribution](https://github.com/nodejs/community-committee/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+contribution%22).
+
+## Teams and Working Groups
+The Community Committee represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. Node.js as a whole needs a broad range of skills and contributions, which are not necessarily technical in nature, to thrive.
+
+You can see the current teams and working groups in the ["Teams and Working Groups"]() section of the README.md file - if there is one that seems to cover an area of particular interest to you, you should take a look at the Team or Working Group's repo to see how you can get further involved.
```
Comment (gr2m) [inline CONTRIBUTING.md]: The link is missing for `Teams and Working Groups`

**Comment by bnb** [comment]
@gr2m Updated that - thanks! This PR predated that being linked, and I added the placeholder so I'd go and add in a section (see https://github.com/nodejs/community-committee/pull/73) 😅

**Comment by nebrius** [comment]
Looks like this is ready to go, thanks again @bnb! 🎉
